docs/SETTINGS.rst

```diff
@@ -88,6 +88,7 @@ Most of the parameters are optional, but you have to specify one of the **host**
 -  **cert**: (optional) file with the client certificate
 -  **key**: (optional) file with the client key. Can be empty if the key is part of **cert**.
 -  **dc**: (optional) Datacenter to communicate with. By default the datacenter of the host is used.
+-  **consistency**: (optional) Select consul consistency mode. Possible values are `default`, `consistent`, or `stale` (more details in https://www.consul.io/api/features/consistency.html)
```

Comment:
Somehow formatting doesn't look consistent (`default` vs ``default``), the link could also be beautified.

Reply:
Thank you for the suggestions, I included them in https://github.com/zalando/patroni/pull/1094/commits/e215c6cc01e7d8487f63160876ea4b4f3ff43608